components/training/eventBooking.tsx

```diff
@@ -167,6 +168,24 @@ const addRightBorder = (index) => {
   return (index + 1) % 3 != 0 ? "md:border-r-8" : "";
 };
 
+const EventDates = ({ duration, date }) => {
+  const startDate = dayjs(date);
+  const endDate = startDate.add(duration, "day");
+
+  const eventDateRange = () => (
+    <>
+      {startDate.format("Do")} - {endDate.format("Do MMMM YYYY")}
+      <div>
+        ({startDate.format("ddd")} - {endDate.format("ddd")})
+      </div>
+    </>
+  );
+
+  const eventDate = () => startDate.format("Do (ddd) MMMM YYYY");
+
+  return <>{duration === 1 ? eventDate() : eventDateRange()}</>;
```

Comment:
given that the above functions return fragments, do you need to wrap them in another fragment for return purposes?

Reply:
To improve the readability, I created those fragments, however, we can do it within the same function if you want?

Reply:
it looks odd thats all - would've expected to return the result of the funcs or return a fragment that has a ternary inside... not both

Reply:
Right, I just moved it in the single function